### PR TITLE
Git: Update bug report template

### PR DESCRIPTION
**Pull request by Erin-Cecele**
Thank you for contributing to Quire! Please complete the form below to submit your pull request for review. 

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Fix, Feature, Docs, or Chore. Issue-# is only needed if this pull request addresses an [exisiting issue](https://github.com/thegetty/quire/issues).*

### Checklist 

Please put an `X` within the brackets that apply `[X]`. 

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file

- [X] I have made my changes in a new branch and not directly in the main branch

- [X] This pull request is ready for final review by the Quire team

### Is this pull request related to an open issue? If so, what is the issue number?

No

### Please describe the changes you made, and call out any details you think are particularly relevant for the Quire team to note in their review.

Update the bug report form per feedback from the group. 

### Does this pull request necessitate changes to Quire's documentation?

No

### Additional Comments

The only way I can see if the changes worked properly in a GitHub Issue Template is by merging the PR, so I made need to do a few of these before it looks correct.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -17,44 +17,57 @@ body:
...
   attributes:
-    label: What browsers are you seeing the problem on?
-    multiple: true
-    options:
-    - Firefox
-    - Chrome
-    - Safari
-    - Microsoft Edge
+    label: Browser 
+    description: Let us know what Browser you are working on (Chrome, Safari, Firefox, Microsoft Edge)? Please include version number.
```
Comment (mphstudios) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:68]:
```suggestion
    description: Let us know what web browser you are working on (Chrome, Safari, Firefox, Microsoft Edge)? Please include the browser version number. 
```

**Comment by Erin-Cecele** [comment]
@mphstudios I need your advice. I noticed there is a YAML syntax error due to the use of the colon in `labels: [status:triage needed]`. The function of this YAML value is to automatically apply the "status:triage needed" label instead of the way we have it currently set up to auto apply the "bug" label. It looks like we might need to rethink the syntax of the "triage" label to remove the colon. What do you think? 

Otherwise, I just made a commit with your suggested corrections.

**Comment by geealbers** [comment]
@Erin-Cecele, does quoting the string fix the error? --> `labels: ['status:triage needed']`

**Comment by Erin-Cecele** [comment]
Yes! That did it. Thank you, @geealbers. Once these errors were resolved, it gave me a preview of the new form. A few more commits are forthcoming to clean things up. I will post when it is ready for final review. Thank you, @mphstudios and @geealbers for your help.

**Comment by Erin-Cecele** [comment]
ok @mphstudios This is now ready for final review.

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 2]
```diff
@@ -11,60 +11,78 @@ body:
...
-    label: What browsers are you seeing the problem on?
-    multiple: true
-    options:
-    - Firefox
-    - Chrome
-    - Safari
-    - Microsoft Edge
+    label: Web Browser 
+    description: What web browser you are working on (Chrome, Safari, Firefox, Microsoft Edge)? Please include the browser version number.
+  validations:
```
Comment (hbalenda) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:69]: I don't think browser needs to be required, for example with a CLI bug the browser wouldn't be relevant

Comment (Erin-Cecele) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:69]: great feedback @hbalenda. I've made that correction!

**Review by mphstudios — APPROVED** [review]

**Comment by Erin-Cecele** [comment]
@mphstudios I just made an update to the bug report form that incorporates the `quire info --debug` command. Let me know if it looks ok. Thank you!

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 3]
```diff
@@ -1,70 +1,72 @@
...
     required: true
 - type: textarea
-  id: version 
+  id: quire-version 
   attributes:
-    label: Version numbers
-    description: Providing version numbers will help us diagnose the issue more quickly. Run `quire --version` to get the Quire version number and `node --version` to get the Node version number. 
-    placeholder: Please include operating system (macOS/Windows), operating system version, Quire version, Node version, and any other version numbers relevant to the issue.
+    label: Version Numbers
+    description: Run `quire info --debug` and list the version numbers for both your **project** (`quire-cli`, `quire-11ty`, `starter`) and your **system** (`quire-cli`, `node`, `npm`, and `os`).
```
Comment (hbalenda) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:44]:
```suggestion
    description: Run `quire info --debug` from your project root to list the version numbers for both your **project** (`quire-cli`, `quire-11ty`, `starter`) and your **system** (`quire-cli`, `node`, `npm`, and `os`).   
```
Edited to clarify where to run the command from - "root" could also be "directory" or whatever you think would make the most sense to users.

Comment (Erin-Cecele) [reply .github/ISSUE_TEMPLATE/bug_report.yaml]: Done. Thank you @hbalenda

**Review by hbalenda — APPROVED** [review]

**Pull request merged**